go/database/log/queries/queries.sql

```diff
@@ -33,3 +33,6 @@ INSERT INTO collection (id, record_enumeration_offset_position, record_compactio
 
 -- name: PurgeRecords :exec
 DELETE FROM record_log r using collection c where r.collection_id = c.id and r.offset <= c.record_compaction_offset_position;
+
+-- name: GetTotalUncompactedRecordsCount :one
+SELECT SUM(record_enumeration_offset_position - record_compaction_offset_position) AS total_uncompacted_depth FROM collection;
```

Comment:
Is there any concern of this query being slow / hogging too much of the node? Maybe we could consider this to move to a read replica if so (in future), since it would be OK with data log.

Reply:
I think it should be ok for now, takes 5-7ms to execute. But good to keep in mind.